certified-connectors/Tilkee-V3/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "ToBeGeneratedOnTilkee",
+          "scopes": [
+            "all"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://app.tilkee.com/v3/oauth/authorize/"
+            },
+            "tokenUrl": {
+              "value": "https://app.tilkee.com/v3/oauth/token/"
+            },
+            "refreshUrl": {
+              "value": "https://app.tilkee.com/v3/oauth/authorize/"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#e44719",
+    "capabilities": [],
+    "publisher": "Flo Sanz"
```

Comment:
@FloxoS The stackOwner property is missing which is a required property. 

Docs - [Add Metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3-add-metadata)